src/DaySlot.jsx

```diff
@@ -78,49 +81,52 @@ let DaySlot = React.createClass({
   },
 
   render() {
-    let {
-        min, max, step, start, end
-      , selectRangeFormat, culture, ...props } = this.props;
-
-    let totalMin = dates.diff(min, max, 'minutes')
-    let numSlots = Math.ceil(totalMin / step)
-    let children = [];
-
-    for (var i = 0; i < numSlots; i++) {
-      children.push(
-        <div key={i} className='rbc-time-slot'/>
-      )
-    }
-
-    this._totalMin = totalMin;
+    const {
+      min,
+      max,
+      step,
+      slices,
+      now,
+      selectRangeFormat,
+      culture,
+      ...props
+    } = this.props
+    this._totalMin = dates.diff(min, max, 'minutes')
 
     let { selecting, startSlot, endSlot } = this.state
-       , style = this._slotStyle(startSlot, endSlot, 0)
+      , style = this._slotStyle(startSlot, endSlot, 0)
 
     let selectDates = {
       start: this.state.startDate,
       end: this.state.endDate
     };
 
     return (
-      <div {...props} className={cn('rbc-day-slot', props.className)}>
-        { children }
-        { this.renderEvents(numSlots, totalMin) }
+      <div {...props} className={cn('rbc-day-slot', this.props.className)}>
+        <TimeGutter
```

Comment:
I wish we could avoid the new extra div here...

One thought is to allow TimeGutter to accept children like:

```jsx
<TimeGutter>
  {renderEvents()}
  {renderSelection()}
</TimeGutter>
```

That avoids the extra div but may not be semantically correct, what do you think?

the reason I bring this up is that the extra div breaks the styling, causing the Gutters to not extpan to their "intrinsic height" which I think can also be fixed with some style tweaks as well if the extra div feels more correct